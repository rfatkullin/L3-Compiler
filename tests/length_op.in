func main (args->[[char]]) -> int
	a -> int;
	arr = [[int] 3] -> [[int]];

	print length(args);

	print length(arr);

	arr[0] = [int 1];
	arr[1] = [int 20];
	arr[2] = [int 100];

	print length(arr[0]) (length(arr[1]) length(arr[2]));

	main = 0;
endfunc
