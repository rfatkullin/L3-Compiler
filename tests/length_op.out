0
3
2000
